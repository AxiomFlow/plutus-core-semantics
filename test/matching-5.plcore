(module Foo (imported) (exported () ()) (local () ())
  (define result
    (case (con Nat.Pair 1 2)
      (cl Nat.Pair (x y) (con Nat.Pair y x)))))
