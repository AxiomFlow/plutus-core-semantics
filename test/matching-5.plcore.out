<T> <k> . </k> <env> ( Foo . result ) |-> ( con Nat . Pair 2  ( 1  .ValList ) ) </env> <mod> Foo </mod> </T>
